cpp/include/models/jni_renderer_skia.hpp

```diff
@@ -51,34 +55,27 @@ namespace rive_android
 		    RenderingStats(20 /* number of samples to average over */
 		    );
 
-		typedef void* (*fp_ATrace_beginSection)(const char* sectionName);
-		typedef void* (*fp_ATrace_endSection)(void);
-		typedef void* (*fp_ATrace_isEnabled)(void);
-
-		void* (*ATrace_beginSection)(const char* sectionName);
-		void* (*ATrace_endSection)(void);
-		void* (*ATrace_isEnabled)(void);
 
 		jobject mKtRenderer;
 
 		SkCanvas* mGpuCanvas;
 		rive::SkiaRenderer* mSkRenderer;
+		ITracer* mTracer;
 
 	public:
-		JNIRendererSkia(jobject ktObject) :
+		JNIRendererSkia(jobject ktObject, bool trace=false) :
```

Comment:
I added `.clang-format` in the project root folder, you might be able to just run the formatter via VSCode to fix some minor styles here.

Reply:
The shortcut is `Alt+Shift+F`, there's also the option in the command palette to `Format Document`